app/src/main/java/com/hmdlong14/cryptocurrency/utils/base/BaseActivity.kt

```diff
@@ -0,0 +1,22 @@
+package com.hmdlong14.cryptocurrency.utils.base
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import androidx.viewbinding.ViewBinding
+
+abstract class BaseActivity : AppCompatActivity() {
+    lateinit var binding: ViewBinding
+
```

Comment:
```suggestion
    private lateinit var _binding: ViewBinding
    protected val binding get() = _binding

```
mình sẽ dùng biến binding này để gọi chứ ko dùng _binding ,_binding chỉ để gán nội bộ trong BaseActivity thôi , tránh sự modify từ bên ngoài